.gitignore

```diff
@@ -6,9 +6,12 @@
 results/
 
 # BeIR datasets folder
-
 datasets/
 
+# Custom dataset files
+syntec.json
+syntec.xlsx
+
```

Comment:
Maye you can delete this ?

Reply:
Done (i wanted to keep them locally, but now everything is on HF I actually will never use local files again...)